modules/ROOT/pages/apikit-asyncapi-module-reference.adoc

```diff
@@ -0,0 +1,236 @@
+
+== Operations
+
+[[Publish]]
+== Publish
+`<apikit-asyncapi:publish>`
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use | None | *Yes* 
+| Content | Binary | None | #[payload] | No  
+| Config Ref | ConfigurationProvider | Name of the configuration used to execute this component | None | *Yes* 
+| Channel Name | String | Channel name | None | *Yes* 
+| Server Name | String | Server name | None | *Yes* 
+| Target Variable | String | Name of a variable on which the operation's output is placed | None | No 
+| Target Value | String | Expression that is evaluated against the operation's output and the effect of that expression is stored in the target variable | #[payload] | No
+| Error Mappings | Array of <<ErrorMapping>> | Set of error mappings | None | No
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+| *Type* | PublishResponse
+| *Attributes Type* | Any
+|===
+
+=== For Configurations
+
+* <<Config>>  
+
+=== Throws
+
+* APIKIT-ASYNCAPI:ASYNCAPI_KIT_INTERNAL_SERVER_ERROR  
+* APIKIT-ASYNCAPI:INTERNAL_SERVER_ERROR
+* APIKIT-ASYNCAPI:PUBLISHING_ERROR
+* APIKIT-ASYNCAPI:VALIDATION_ERROR
+
+== Sources
+
+[[MessageListener]]
+== Message Listener
+`<apikit-asyncapi:message-listener>`
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use | None | *Yes* 
+| Channel Name | String | Channel name | None | *Yes* 
+| Servers | Array of String | List of servers to listen to | None | No 
+| Output MIME Type | String | MIME type of the payload that this operation outputs | None | No 
+| Output Encoding | String | Encoding of the payload that this operation outputs | None | No  
+| Primary Node Only | Boolean | Specifies if this source must be executed on the primary node when running in Cluster | None | No 
+| Config Ref | ConfigurationProvider | Name of the configuration used to execute this component | None | *Yes* 
+| Streaming Strategy a| * <<RepeatableInMemoryStream>>
+* <<RepeatableFileStoreStream>>
+* non-repeatable-stream | Set up whether to use repeatable streams and define their behavior | None | No
+
+| Redelivery Policy | <<RedeliveryPolicy>> | Policy for processing the redelivery of the same message | None | No
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+| *Type* | Any
+| *Attributes Type* a| One of:
+
+* <<KafkaMessageAttributes>>
+* <<AnypointMqMessageAttributes>>
+|===
+
+=== For Configurations
+
+* <<Config>>  
+
+== Types
+
+[[KafkaConfig]]
+=== Kafka Config
+
+[cols=".^20%,.^25%,.^30%,.^15%,.^10%", options="header"]
+|======================
+| Field | Type | Description | Default Value | Required
+| Server Key | String | Identifier for the server configuration within the AsyncAPI configuration | None | *Yes*
+| Producer Config Ref | String | Reference to the Kafka producer configuration | None | No
```

Comment:
Should producer and consumer be init cap?

Reply:
Globally.

Reply:
You're right! 
Thanks!